compiler/testData/debug/localVariables/suspend/underscoreNames.kt

```diff
@@ -17,36 +17,50 @@ suspend fun box() = foo(A()) { (x_param, _, y_param) ->
 // However, in non-IR, the fields are first stored in local variables, and they are not read directly (even for destructuring components).
 // In IR, the fields are directly read from.
 
-// METHOD : UnderscoreNamesKt$test$2.invokeSuspend(Ljava/lang/Object;)Ljava/lang/Object;
+// The local variable for destructuring lambda arguments, in this case
+// `$dstr$x_param$_u24__u24$y_param`, is moved to a field in the IR backend,
```

Comment:
Is it right only for suspend lambdas? Or for general one too?

Reply:
Will find a definitive answer, but I believe it's in general.

Reply:
Nope, just suspend functions. Will adjust the note. Also, turns out there are discrepancies in plain lambdas and destructuring as well! Next PR ... :)